docs/source/en/tasks/prompting.md

```diff
@@ -0,0 +1,466 @@
+... Sentiment:
+... """
+
+>>> sequences = pipe(
+...     prompt,
+...     max_length=100,
+...     do_sample=True,
+...     top_k=10,
+...     num_return_sequences=1,
+...     eos_token_id=tokenizer.eos_token_id,
+... )
+
+>>> for seq in sequences:
+...     print(f"Result: {seq['generated_text']}")
+Result: Classify the text into neutral, negative or positive. 
+Text: This movie is definitely one of my favorite movies of its kind. The interaction between respectable and morally strong characters is an ode to chivalry and the honor code amongst thieves and policemen.
+Sentiment:
+Positive
+```
+
+As a result, the output contains a classification label from the list we have provided in the instructions, and it is a correct one!
+
+<Tip>
+
+You may notice that in addition to the prompt, we pass a number of text generation parameters. We cover these parameters 
+in other docs, such as the [Text generation strategies](../generation_strategies) guide. For the purposes of this guide, 
+you only need to know that  `max_length` controls the length of the total output (including the initial prompt). 
+</Tip>
+
+#### Named Entity Recognition
+
+Named Entity Recognition (NER) is a task of finding named entities in a piece of text, such as a person, location, or organization.
+Let's modify the instructions in the prompt to make the LLM perform this task. Here, let's also set `return_full_text = False` 
+so that output doesn't contain the prompt:
+
+```python
+>>> torch.manual_seed(1) # doctest: +IGNORE_RESULT
+>>> prompt = """Return a list of named entities in the text.
+... Text: The Golden State Warriors are an American professional basketball team based in San Francisco.
+... Named entities:
+... """
+
+>>> sequences = pipe(
+...     prompt,
+...     max_length=43,
+...     do_sample=True,
+...     top_k=10,
+...     num_return_sequences=1,
+...     return_full_text = False,    
+...     eos_token_id=tokenizer.eos_token_id,
+... )
+
+>>> for seq in sequences:
+...     print(f"{seq['generated_text']}")
+- The Golden State Warriors
+- San Francisco
+```
+
+As you can see, the model correctly identified two named entities from the given text.
+
+#### Translation
+
+Another task LLMs can perform is translation. You can choose to use encoder-decoder models for this task, however, here,
+for the simplicity of the examples, we'll keep using Falcon-7b-instruct, which does a decent job. Once again, here's how 
+you can write a basic prompt to instruct a model to translate a piece of text from English to Italian: 
+
+```python
+>>> torch.manual_seed(2) # doctest: +IGNORE_RESULT
+>>> prompt = """Translate the English text to Italian.
+... Text: Sometimes, I've believed as many as six impossible things before breakfast.
+... Translation:
+... """
+
+>>> sequences = pipe(
+...     prompt,
+...     max_length=100,
+...     do_sample=True,
+...     top_k=10,
```

Comment:
Ok to use sampling here for me as translation outputs a longer sequence, multiple answers are correct and do_sample=True can help here